pkg/libvirt/libvirt.go

```diff
@@ -88,7 +88,7 @@ func (d *Driver) setVcpus(cpus uint) error {
 		return err
 	}
 
-	d.CPU = int(cpus)
+	d.CPU = cpus
```

Comment:
Most likely introduced by the update of `crc-org/machine`.

Reply:
Yeah, right.